internal/services/costmanagement/export_resource_group_resource_test.go

```diff
@@ -127,14 +127,16 @@ resource "azurerm_resource_group_cost_management_export" "test" {
   recurrence_type              = "Monthly"
   recurrence_period_start_date = "%sT00:00:00Z"
   recurrence_period_end_date   = "%sT00:00:00Z"
+  partition_data_enabled               = true
```

Comment:
The basic test should only include properties that are `Required`, since this is an `Optional` property this shouldn't be here 
```suggestion
```